### PR TITLE
Extend main README with a "Who is Apollo?" section

### PR DESCRIPTION
**Pull request by daniman**
We're planning to add a "Who is Apollo" section to most of our project READMEs, so this is the first of  few similar commits across our different projects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,15 @@ The Apollo Client API reference can be found at: <br/>
 - [@benjamn](https://github.com/benjamn) (Apollo)
 - [@hwillson](https://github.com/hwillson) (Apollo)
 - [@jcreighton](https://github.com/jcreighton) (Apollo)
+- [@brainkim](https://github.com/brainkim) (Apollo)
+
+### Who is Apollo?
```
Comment (hwillson) [inline README.md:26]: We might want to change this to a `##` instead, to call it out a bit more.

File: README.md [hunk 2]
```diff
@@ -21,3 +21,15 @@ The Apollo Client API reference can be found at: <br/>
 - [@benjamn](https://github.com/benjamn) (Apollo)
 - [@hwillson](https://github.com/hwillson) (Apollo)
 - [@jcreighton](https://github.com/jcreighton) (Apollo)
+- [@brainkim](https://github.com/brainkim) (Apollo)
```
Comment (hwillson) [inline README.md:24]: Ooops - great catch! 😳 (sorry for not adding you sooner Brian!)

**Review by hwillson — APPROVED** [review]
This is awesome @daniman! Just one comment about the title, but the content is all 👍 for sure!

File: README.md (hunk 1 above)
Comment (daniman) [reply README.md:26]: oh whoops, good call

**Pull request merged**